nservicebus/rabbitmq/configuration-api.md

```diff
@@ -14,197 +14,242 @@ To use RabbitMQ as the underlying transport:
-
-snippet:rabbitmq-config-connectionstringname
+NOTE: For debugging purposes, it can be helpful to increase the `RequestedHeartbeat` and `DequeueTimeout` settings like this:
 
-To specify the connection string in code:
+snippet:rabbitmq-connectionstring-debug
 
-snippet:rabbitmq-config-connectionstring-in-code
 
-For debugging purposes, increase the `RequestedHeartbeat` and `DequeueTimeout` like this:
+## Callback support
 
-snippet:rabbitmq-connectionstring-debug
+When scaling out an endpoint using the RabbitMQ transport, each instance of the endpoint will consume messages from the same shared broker queue. This works in most scenarios because any instance can handle any incoming message. However, this is not true for callback messages, which need to be handled by the instance that requested the callback. 
 
 
-### Callback support
+### Versions 3 and below
 
-RabbitMQ is a broker which means that scale out is done by adding more endpoints feeding of the same broker queue. This usually works fine if no state is shared between the different instances. This is not the case for callbacks since they do rely on local state kept in memory. In order to seamlessly support this scenario out of the box the RabbitMQ transport has the concept of a callback receiver. Essentially this is a separate queue named `{endpointname}.{machinename}` to which all callbacks are routed. This means that callbacks are handled by the same instance that requested them. This behavior is on by default. If not using callbacks it can disable it using the following configuration:
+In Versions 3 and below, callbacks are enabled by default, and the transport will create a separate callback receiver queue, named `{endpointname}.{machinename}`, to which all callbacks are routed. If callbacks are not being used, the callback receiver can be disabled using the following setting:
 
 snippet:rabbitmq-config-disable-callback-receiver
 
-NOTE: In Versions 4 and above Callbacks are disabled by default. To enable them follow [Callbacks documentation](/nservicebus/messaging/handling-responses-on-the-client-side.md#message-routing-version-6-and-above).
-
 This means that the queue will not be created and no extra threads will be used to fetch messages from that queue.
 
-By default 1 dedicated thread is used for the callbacks. To add more threads, due to a high rate of callbacks, use the following:
+By default, 1 dedicated thread is used for the callbacks. To add more threads, due to a high rate of callbacks, use the following:
 
 snippet:rabbitmq-config-callbackreceiver-thread-count
 
 
-### Controlling the message ID strategy
+### Versions 4 and above
+
+In Versions 4 and above, callbacks no longer directly managed by the RabbitMQ transport and are not enabled by default. To enable them, follow the  [Callbacks documentation](/nservicebus/messaging/handling-responses-on-the-client-side.md#message-routing-nservicebus-callbacks-version-1-and-above).
```

Comment:
minor tweak: added `follow the steps outlined in` to make it read better. 

In Versions 4 and above, callbacks no longer directly managed by the RabbitMQ transport and are not enabled by default. To enable them, follow the steps outlined in the [Callbacks documentation](/nservicebus/messaging/handling-responses-on-the-client-side.md#message-routing-nservicebus-callbacks-version-1-and-above).